doc/api/addons.md

```diff
@@ -965,6 +965,10 @@ provided by the underlying V8 JavaScript engine. They are subject to change
 or removal at any time. They are not documented by Node.js or V8, and they
 should never be used outside of testing.
 
+During shutdown of the process or worker threads destructors are not called
+by the JS engine. Therefore it's the responsibility of the user to track
+these objects and ensure proper destruction to avoid resource leaks.
+
```

Comment:
I believe this case is handled for addons using Node-API. Would that be worth mentioning?

Reply:
Not sure if this would provide any benefit. When someone ends up reading the doc of ObjectWrap till the end it's quite clear that this person is deep in the non Napi area for whatever reason.

Reply:
ok, just thought I'd ask.